e2e/scenario_win_test.go

```diff
@@ -40,12 +42,14 @@ func Test_Windows2022Containerd(t *testing.T) {
 			BootstrapConfigMutator: func(configuration *datamodel.NodeBootstrappingConfiguration) {
 			},
 			Validator: func(ctx context.Context, s *Scenario) {
-				ValidateWindowsVersionFromWindowsSettings(ctx, s, "2022-containerd")
-				ValidateWindowsProductName(ctx, s, "Windows Server 2022 Datacenter")
-				ValidateWindowsDisplayVersion(ctx, s, "21H2")
-				ValidateFileHasContent(ctx, s, "/k/kubeletstart.ps1", "--container-runtime=remote")
-				ValidateWindowsProcessHasCliArguments(ctx, s, "kubelet.exe", []string{"--rotate-certificates=true", "--client-ca-file=c:\\k\\ca.crt"})
-				ValidateCiliumIsNotRunningWindows(ctx, s)
+				if config.Config.SSHEnabled {
```

Comment:
With SSH disabled, I think that you can exec into a system container (e.g. csi pod) to run the check. Otherwise, there is no validation.